### ISSUE TITLE
api: account endpoint should not be public

### ISSUE THREAD

**Issue by fsouza** [segue-org/segue#1]
Currently, it exposes sensitive data about users (like the phone number and the CPF number). Example:

```
{
  "resource": {
    "$type": "Account.normal", 
    "city": "Rio de Janeiro", 
    "country": "Brazil", 
    "created": "Thu, 26 Feb 2015 22:30:17 GMT", 
    "email": "franciscossouza@gmail.com", 
    "id": 15, 
    "last_updated": "Fri, 27 Feb 2015 11:15:52 GMT", 
    "name": "Francisco Souza", 
    "organization": "Globo.com", 
    "phone": "WAT?!", 
    "resume": "Core developer do tsuru, PaaS open source; fanboy de concorr\u00eancia e paralelismo."
  }
}
```

It's also a nice source of email addresses for spammers :-(

**Comment by tarsisazevedo** [comment]
+1

**Comment by fmobus** [comment]
Thanks for the report. Indeed, the endpoint was not requiring authentication. We took the system offline whilst we apply and further test the patch.

**Comment by fmobus** [comment]
Pull request was merged into master, but we had to some more fixes with commit 793f8e0. Should do the trick now.
